kqueen/config/demo.py

```diff
@@ -3,7 +3,7 @@
 
 class Config(BaseConfig):
     DEBUG = False
-    LOG_LEVEL = 'INFO'
+    LOG_LEVEL = 'DEBUG'
```

Comment:
Log level is set in logging config. This option is unnecessary.

Reply:
Also, changes in your commit are not logically connected, so it may be better to separate ones anyway

Reply:
yep, sure